src/sentry/models/project.py

```diff
@@ -110,7 +110,7 @@
     "minidump",
     "native",
     "native-qt",
-    "nintendo",
+    "nintendo-switch",
```

Comment:
did you leave this in here by accident 😅

Reply:
no I ended up not merging my backend change since the only line needed from it was this one 😅 It's fine. It's just used for type checking in getsentry.